src/main/java/me/markng/uhcdatapresenter/mixin/HeaderFooter.java

```diff
@@ -12,10 +12,10 @@
 public class HeaderFooter {
 	@Inject(method = "onPlayerListHeader(Lnet/minecraft/network/packet/s2c/play/PlayerListHeaderS2CPacket;)V", at = @At("HEAD"))
 	public void onPlayerListHeader(PlayerListHeaderS2CPacket packet, CallbackInfo ci) {
-		if (!packet.getHeader().getString().isEmpty())
-			DataAPI.sendMessage("{\"header\":\"" + packet.getHeader().getString().replace("\n", "<br>") + "\"}");
-		if (!packet.getFooter().getString().isEmpty())
-			DataAPI.sendMessage("{\"footer\":\"" + packet.getFooter().getString().replace("\n", "<br>") + "\"}");
+		if (!packet.header().getString().isEmpty())
+			DataAPI.sendMessage("{\"header\":\"" + packet.header().getString().replace("\n", "<br>") + "\"}");
+		if (!packet.footer().getString().isEmpty())
+			DataAPI.sendMessage("{\"footer\":\"" + packet.header().getString().replace("\n", "<br>") + "\"}");
```

Comment:
Just looked through my own commit again and saw this mistake 🤦

```suggestion
			DataAPI.sendMessage("{\"footer\":\"" + packet.footer().getString().replace("\n", "<br>") + "\"}");
```